system/jlib/jptree.cpp

```diff
@@ -9187,3 +9187,4 @@ jlib_decl IPropertyTree * queryCostsConfiguration()
 {
     return queryComponentConfig().queryPropTree("costs");
 }
+
```

Comment:
Extra newline - which should have been caught by a hook preventing newlines at the end.